app/globals.py

```diff
@@ -101,3 +105,9 @@ def get_metadata(user: User) -> Union[None, MappingProxyType]:
 def get_answer_store(user: User) -> AnswerStore:
     questionnaire_store = get_questionnaire_store(user.user_id, user.user_ik)
     return questionnaire_store.answer_store
+
+
+def get_view_submitted_response_expired(submitted_at: datetime) -> bool:
```

Comment:
As this method return a boolean, it would be clearer if it was named `is_` rather than `get_`
```suggestion
def is_view_submitted_response_expired(submitted_at: datetime) -> bool:
```

Something else we should consider, probably outside of this pull request, is where we put shared code like this. We should review the use of globals.py and the helpers, libs and utlities namespaces.